internal/bzlmod/go_mod.bzl

```diff
@@ -120,8 +122,16 @@ def _parse_directive(state, directive, tokens, comment, path, line_no):
             version = tokens[1],
             direct = comment != "indirect",
         ))
-
-    # TODO: Handle exclude and replace.
+    elif directive == "replace":
+        from_path = tokens[0]
+        if tokens[1] != "=>":
```

Comment:
Please also check the expected length of `tokens` here so that users see the error message, not a generic out of bounds error.

Reply:
Updated. Good idea!